docroot/themes/custom/uids_base/templates/content/node.html.twig

```diff
@@ -105,6 +105,5 @@
 
   <div{{ content_attributes.addClass('node__content') }}>
     {{ content }}
-  </div>
```

Comment:
Just scanning changes - was this intentional?

Reply:
Cool. I'm assuming Percy would've caught that but we're out of snapshots. :(

Reply:
Is there a way to select when we run percy? So then it doesn't automatically run after every push? Sorry if naive/obvious question... or if it's impossible jk

Reply:
can we switch this PR to draft if it isn't ready?

Reply:
I think it is, thats why I'm reviewing and testing it.